charts/ocis/templates/postprocessing/jobs.yaml

```diff
@@ -0,0 +1,111 @@
+{{- if .Values.services.postprocessing.maintenance.restartPostprocessingFinished.enabled }}
+{{- include "ocis.basicServiceTemplates" (dict "scope" . "appName" "appNamePostprocessing" "appNameSuffix" "") -}}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: postprocessing-restart-finished
+  namespace: {{ template "ocis.namespace" . }}
+  labels:
+    {{- include "ocis.labels" . | nindent 4 }}
+spec:
+  schedule: "{{ .Values.services.postprocessing.maintenance.restartPostprocessingFinished.schedule }}"
+  successfulJobsHistoryLimit: 3
+  failedJobsHistoryLimit: 3
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      parallelism: 1
+      template:
+        metadata:
+          labels:
+            app: postprocessing-restart-finished
+            {{- include "ocis.labels" . | nindent 12 }}
+
+        spec:
+          restartPolicy: Never
+          {{- include "ocis.hostAliases" $ | nindent 10 }}
+          nodeSelector: {{ toYaml $.jobNodeSelector | nindent 12 }}
+          containers:
+            - name: postprocessing-restart-finished
+              {{- include "ocis.jobContainerImageOcis" . | nindent 14 }}
+              command: ["ocis"]
+              args: ["postprocessing", "restart", "-s", "finished"]
+              securityContext:
+                runAsNonRoot: true
+                runAsUser: {{ .Values.securityContext.runAsUser }}
+                runAsGroup: {{ .Values.securityContext.runAsGroup }}
+                readOnlyRootFilesystem: true
+              env:
+                {{- include "ocis.serviceRegistry" . | nindent 16 }}
+                {{- include "ocis.events" . | nindent 16 }}
+                {{- include "ocis.persistentStore" . | nindent 16 }}
```

Comment:
The command only needs the `ocis.events` configuration. It does not need access to the `serviceRegistry` and the `persistentStore`

Reply:
great find, fixed it :+1: